cloud_experts_tutorials/cloud-experts-using-cloudfront-and-waf.adoc

```diff
@@ -24,7 +24,7 @@ You can use an Amazon CloudFront to add a Web Application Firewall (WAF) to your
 [id="prerequisites_{context}"]
 == Prerequisites
 
-* xref:../rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc#rosa-sts-creating-a-cluster-quickly[A ROSA Classic cluster].
+* A ROSA (HCP or Classic) cluster.
```

Comment:
```suggestion
* A ROSA or ROSA (classic architecture) cluster.
```

Reply:
See above comment about keeping Classic and HCP until we publish that across the docs.